app/models/agents/lifx_pulse_lights_agent.rb

```diff
@@ -0,0 +1,64 @@
+module Agents
+  class LifxPulseLightsAgent < Agent
```

Comment:
I'd definitely prefer having one Agent that's more flexible.  Maybe we need to extend FormConfigurable?